tests/CLI/modules/order_tests.py

```diff
@@ -309,7 +309,8 @@ def test_location_list(self):
         result = self.run_command(['order', 'package-locations', 'package'])
         self.assert_no_fail(result)
         expected_results = [
-            {'id': 2017603, 'dc': 'wdc07', 'description': 'WDC07 - Washington, DC', 'keyName': 'WASHINGTON07'}
+            {'id': 2017603, 'dc': 'wdc07', 'description': 'WDC07 - Washington, DC',
+             'keyName': 'WASHINGTON07', 'note': 'closed soon: wdc07.pod01'}
```

Comment:
```suggestion
             'keyName': 'WASHINGTON07', 'Note': 'closed soon: wdc07.pod01'}
```

Also, please remove the debugging code accidently committed whenever I made this test